frontend/src/gallery/gallery.api.ts

```diff
@@ -2,9 +2,14 @@ import { BASE_URL } from '../data/constants.js';
 import { ImagesResponse } from '../interfaces/response';
 
 export class GalleryApi {
-  async fetchImages(pageNumber: number, pageLimit: number): Promise<ImagesResponse> {
+  async fetchImages(pageNumber: number, pageLimit: number, user?: string): Promise<ImagesResponse> {
     const accessToken = localStorage.getItem('token');
-    const url = `${BASE_URL}/gallery?page=${pageNumber}&limit=${pageLimit}`;
+
+    let url = `${BASE_URL}/gallery?page=${pageNumber}&limit=${pageLimit}`;
```

Comment:
you can refactor it like this:
let url =  `${BASE_URL}/gallery?page=${pageNumber}`
url = user ? `${url}&filter=${user}` : url;